integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/AllDataTypesTestCase.scala

```diff
@@ -73,6 +75,21 @@ class AllDataTypesTestCase extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("insert data with from_unixtime(0) and query") {
+    sql("drop table if exists time_carbon1")
+    sql("create table if not exists time_carbon1(time1 timestamp) stored as carbondata")
+    sql("insert into time_carbon1 select from_unixtime(0)")
```

Comment:
insert into table select '', results should be NULL. After this change, it is inserting default time. Please check

Reply:
ok